src/Scouting.elm

```diff
@@ -30,18 +31,25 @@ type alias Model =
     { scouter : String
     , team : Maybe Int
     , match : Maybe Int
-    , driverStation : String
+    , driverStation : String --PossibleStations
     , isStarted : Bool
     }
 
 
-driverStations : List (List Int)
-driverStations =
-    [ [ 1690, 1574, 3339, 254, 2056, 1323 ]
-    , [ 118, 1577, 1024, 2056, 1690, 254 ]
-    , [ 1574, 3339, 1577, 1323, 1024, 118 ]
-    , [ 3339, 1574, 1577, 1024, 118, 2056 ]
-    ]
+
+{- }
+   PossibleStations =
+       "כחול 1"
+       | "כחול 2"
+       | "כחול 3"
+       |"אדום 1"
+       |"אדום 2"
+       | "אדום 3"
+-}

```

Comment:
Did you mean
```elm
type PossibleStations
    = Blue1
    | Blue2
    | Blue3
    | Red1
    | Red2
    | Red3
```
?
You can't create a type from literal strings like this.

Reply:
Yeah, in the end I did it properly I think